Flake starting control plane in TestAssertExpressions

From an attempt with `-count 100`:

```
2025-11-19T07:51:04.9054760Z === RUN   TestAssertExpressions
2025-11-19T07:51:04.9054855Z     expression_integration_test.go:46: 
2025-11-19T07:51:04.9055233Z            Error Trace:    /home/runner/work/kuttl/kuttl/internal/step/expression_integration_test.go:46
2025-11-19T07:51:04.9055414Z            Error:          Received unexpected error:
2025-11-19T07:51:04.9056409Z                            unable to start control plane itself: failed to start the controlplane. retried 5 times: timeout waiting for process kube-apiserver to start successfully (it may have failed to start, or stopped unexpectedly before becoming ready) 
2025-11-19T07:51:04.9056603Z            Test:           TestAssertExpressions
2025-11-19T07:51:04.9056698Z --- FAIL: TestAssertExpressions (216.32s)
2025-11-19T07:51:04.9056916Z panic: runtime error: invalid memory address or nil pointer dereference [recovered]
2025-11-19T07:51:04.9057088Z    panic: runtime error: invalid memory address or nil pointer dereference
2025-11-19T07:51:04.9057255Z [signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x28da73b]
2025-11-19T07:51:04.9057264Z     
2025-11-19T07:51:04.9057445Z goroutine 23919 [running]:
2025-11-19T07:51:04.9057562Z testing.tRunner.func1.2({0x2b3c4a0, 0x43be610})
2025-11-19T07:51:04.9064121Z    /home/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.24.0.linux-amd64/src/testing/testing.go:1734 +0x3eb
2025-11-19T07:51:04.9064223Z testing.tRunner.func1()
2025-11-19T07:51:04.9064691Z    /home/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.24.0.linux-amd64/src/testing/testing.go:1737 +0x696
2025-11-19T07:51:04.9064805Z panic({0x2b3c4a0?, 0x43be610?})
2025-11-19T07:51:04.9065144Z    /home/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.24.0.linux-amd64/src/runtime/panic.go:787 +0x132
2025-11-19T07:51:04.9065374Z github.com/kudobuilder/kuttl/internal/step.TestAssertExpressions(0xc00094c700)
2025-11-19T07:51:04.9065643Z    /home/runner/work/kuttl/kuttl/internal/step/expression_integration_test.go:91 +0x85b
2025-11-19T07:51:04.9065734Z testing.tRunner(0xc00094c700, 0x2f49ca0)
2025-11-19T07:51:04.9066436Z    /home/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.24.0.linux-amd64/src/testing/testing.go:1792 +0x226
2025-11-19T07:51:04.9066614Z created by testing.(*T).Run in goroutine 1
2025-11-19T07:51:04.9067007Z    /home/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.24.0.linux-amd64/src/testing/testing.go:1851 +0x8f3
2025-11-19T07:51:04.9067234Z FAIL       github.com/kudobuilder/kuttl/internal/step      1538.867s
```

This might be due to improperly cleaning up envtest instances, as at the end of the run the following was present in the log:
```
2025-11-19T07:51:05.6566896Z Cleaning up orphan processes
2025-11-19T07:51:05.8183441Z Terminate orphan process: pid (16670) (etcd)
2025-11-19T07:51:05.8213486Z Terminate orphan process: pid (16938) (kube-apiserver)
2025-11-19T07:51:05.8241649Z Terminate orphan process: pid (17418) (etcd)
2025-11-19T07:51:05.8274412Z Terminate orphan process: pid (17435) (kube-apiserver)
2025-11-19T07:51:05.8341189Z Terminate orphan process: pid (18909) (etcd)
2025-11-19T07:51:05.8371519Z Terminate orphan process: pid (19184) (kube-apiserver)
2025-11-19T07:51:05.8408609Z Terminate orphan process: pid (20536) (etcd)
2025-11-19T07:51:05.8449611Z Terminate orphan process: pid (20682) (kube-apiserver)
2025-11-19T07:51:05.8496479Z Terminate orphan process: pid (21965) (etcd)
2025-11-19T07:51:05.8531364Z Terminate orphan process: pid (21981) (kube-apiserver)
2025-11-19T07:51:05.8562021Z Terminate orphan process: pid (23844) (etcd)
2025-11-19T07:51:05.8600068Z Terminate orphan process: pid (23861) (kube-apiserver)
2025-11-19T07:51:05.8634600Z Terminate orphan process: pid (25697) (etcd)
2025-11-19T07:51:05.8670434Z Terminate orphan process: pid (25756) (kube-apiserver)
2025-11-19T07:51:05.8721295Z Terminate orphan process: pid (27436) (etcd)
2025-11-19T07:51:05.8770364Z Terminate orphan process: pid (27533) (kube-apiserver)
2025-11-19T07:51:05.8893720Z Terminate orphan process: pid (29152) (etcd)
2025-11-19T07:51:05.8968539Z Terminate orphan process: pid (29409) (kube-apiserver)
2025-11-19T07:51:05.9087680Z Terminate orphan process: pid (31037) (etcd)
2025-11-19T07:51:05.9136259Z Terminate orphan process: pid (31091) (kube-apiserver)
2025-11-19T07:51:05.9257050Z Terminate orphan process: pid (32959) (etcd)
2025-11-19T07:51:05.9290267Z Terminate orphan process: pid (32975) (kube-apiserver)
2025-11-19T07:51:05.9344210Z Terminate orphan process: pid (34354) (etcd)
2025-11-19T07:51:05.9380569Z Terminate orphan process: pid (34440) (kube-apiserver)
2025-11-19T07:51:05.9414714Z Terminate orphan process: pid (35995) (etcd)
2025-11-19T07:51:05.9458735Z Terminate orphan process: pid (36161) (kube-apiserver)
2025-11-19T07:51:05.9500763Z Terminate orphan process: pid (37478) (etcd)
2025-11-19T07:51:05.9539059Z Terminate orphan process: pid (37650) (kube-apiserver)
2025-11-19T07:51:05.9763416Z Terminate orphan process: pid (39453) (etcd)
2025-11-19T07:51:05.9826702Z Terminate orphan process: pid (39490) (kube-apiserver)
2025-11-19T07:51:05.9869776Z Terminate orphan process: pid (41404) (etcd)
2025-11-19T07:51:05.9916806Z Terminate orphan process: pid (41420) (kube-apiserver)
2025-11-19T07:51:06.0010079Z Terminate orphan process: pid (43037) (etcd)
[and more...]
```

--- a/internal/step/expression_integration_test.go
+++ b/internal/step/expression_integration_test.go
@@ -15,6 +15,7 @@ import (
 	kfile "github.com/kudobuilder/kuttl/internal/file"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
@@ -44,7 +45,7 @@ func TestAssertExpressions(t *testing.T) {
 	defer cancel()
 
 	testenv, err := kubernetes.StartTestEnvironment(false)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 
 	codednsDeployment := &appsv1.Deployment{
 		ObjectMeta: metav1.ObjectMeta{

